anomalib/data/mvtec.py

```diff
@@ -24,30 +24,39 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging
-import tarfile
 from pathlib import Path
-from typing import Optional, Tuple, Union
-from urllib.request import urlretrieve
+from typing import Optional, Sequence, Tuple, Union
 
 import albumentations as A
 from pandas import DataFrame
 
 from anomalib.data.base import AnomalibDataModule, AnomalibDataset
 from anomalib.data.task_type import TaskType
 from anomalib.data.utils import (
-    DownloadProgressBar,
+    DownloadInfo,
     InputNormalizationMethod,
     Split,
     TestSplitMode,
     ValSplitMode,
+    download_and_extract,
     get_transforms,
-    hash_check,
 )
 
 logger = logging.getLogger(__name__)
 
 
-def make_mvtec_dataset(root: Union[str, Path], split: Optional[Union[Split, str]] = None) -> DataFrame:
+IMG_EXTENSIONS = [".png", ".PNG"]
+
+DOWNLOAD_INFO = DownloadInfo(
+    name="mvtec",
+    url="https://www.mydrive.ch/shares/38536/3830184030e49fe74747669442f0f282/download/420938113-1629952094",
+    hash="eefca59f2cede9c3fc5b6befbfec275e",
+)
+
+
+def make_mvtec_dataset(
+    root: Union[str, Path], split: Optional[Union[Split, str]] = None, extensions: Sequence[str] = (".png")
```

Comment:
Rather than providing a new default value for `extensions`, can we use `IMG_EXTENSIONS` directly? Something like

```python
    root: Union[str, Path], split: Optional[Union[Split, str]] = None, extensions: Sequence[str] = IMG_EXTENSIONS
```

Reply:
I tried this earlier but some of the code quality checks were complaining